R/ggplot.R

```diff
@@ -10,6 +10,12 @@
 
 ggplot_build_set <- function() {
   if (!is_installed("ggplot2")) return(NULL)
+  # Alternative update method if we're dealing with S7
+  if ("class_ggplot" %in% getNamespaceExports("ggplot2")) {
+    .globals$ggplot_build <- getFromNamespace("build_ggplot", "ggplot2")
```

Comment:
I don't think this is needed since the restore in this case registers `dummy_method`?

```suggestion
```